ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/index.tsx

```diff
@@ -185,6 +212,19 @@ const RunComponent = () => {
                 </TableCell>
               </TableRow>
             )}
+            <TableRow>
+              <TableCell>Logs</TableCell>
+              <TableCell>
+                <Link onClick={() => handleDownload(ortRun.id)}>
+                  <Button
+                    variant='outline'
+                    className='font-semibold text-blue-400'
+                  >
+                    Download ZIP file
+                  </Button>
+                </Link>
```

Comment:
Does the download logic work if this is not a `Link` but a `Button` with the same `onClick`? If it works, this should probably be a `Button`.

Reply:
Using Button only stretches the buttons along the whole column, but that should be fixable.